### ISSUE TITLE
(Similar to previously closed issue) Suspended account

### ISSUE THREAD

**Issue by Raven-621** [KhoalaS/Infinity-For-Reddit#8]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.9.0

### Where did you get the app from

Other

### Android version

10, One UI 2.0

### Current behaviour

Suddenly, i got permanently suspended, after using the latest version for around a day-ish, i think. I think it is indeed because of Infinity, since it's sudden and it's not like i did anything to break Reddit site-wide rules recently, if not ever.

![Screenshot_20230813-223933_Reddit](https://github.com/KhoalaS/Infinity-For-Reddit/assets/141109979/e2f7deec-539d-46f1-b49a-13c3222448b1)

**Comment by KhoalaS** [comment]
Yes seems like they are taking action now. I am working on migrating some stuff to hopefully make it less detectable, but as always cant gaurantee if they ban again.

**Comment by TapuCosmo** [comment]
Same, just got appeal denied too, haven't posted anything in 3 months.

**Comment by ramblinguy** [comment]
Yeah just wanted to add I got a ban too, despite having premium

**Comment by Turhvjbufv** [comment]
They got us good😭

**Comment by mitomenai** [comment]
@KhoalaS I haven't looked at the source code, but it should technically be possible to bypass the need for app API key entirely and use oauth.reddit.com directly with a user oauth token just like the website. Give it a generic user agent like Chrome and it should be fairly undetectable. What do you think?

**Comment by KhoalaS** [comment]
> 

Hm, so basically use the login endpoint but identify as the web version and then get the access token. Could work. But I don't think I will be doing much work with this repo anymore after the last update.

**Comment by coxackie** [comment]
@KhoalaS I hope you are not pulling out because of the Reddit bans. I am sure a lot of people (including yours trully) would like to see this fork develop.

**Comment by Sintinium** [comment]
I was just suspended as well. I was using whichever branch the main release uses.

**Comment by ramblinguy** [comment]
I was the main branch for a bit, and then I used the branch that let you enter your own API key. That's what I was using when I got banned, but they might have already detected me using the main branch, so I can't say which one caused it

**Comment by Sintinium** [comment]
At the very start I used a version that took an API key but a few weeks ago I switched to the reddit login version
